src/dmd/mars.d

```diff
@@ -1653,6 +1653,10 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                 params.cplusplus = CppStdRevision.cpp98;
             else if (strcmp(p + 8, "c++11") == 0)
                 params.cplusplus = CppStdRevision.cpp11;
+            else if (strcmp(p + 8, "c++14") == 0)
+                params.cplusplus = CppStdRevision.cpp14;
+            else if (strcmp(p + 8, "c++17") == 0)
+                params.cplusplus = CppStdRevision.cpp17;
```

Comment:
Would be great to implement a usage subpage that lists all available options like it's done for e.g. `-transition`

Reply:
-> https://github.com/dlang/dmd/pull/9217